### PR TITLE
Fix performance of the global prom stats worker queries

### PR DESCRIPTION
**Pull request by josephschorr**
- Change the counts to use the MySQL information schema for estimates
- Change the lookup of active users to skip the unnecessary heavy filter

### CODE REVIEW

File: data/estimate.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def mysql_estimate_row_count(model_cls, db):
+    """ Uses the information_schema to retrieve the row count for a table. """
+    query = "SELECT table_name, table_rows FROM information_schema.tables WHERE table_schema = DATABASE() AND table_name = %s"
+    cursor = db.execute_sql(query, (model_cls._meta.table_name,))
+    res = cursor.fetchone()
```
Comment (jakedt) [inline data/estimate.py]: will this raise an error if there isn't a result? are you ok with that?

File: data/estimate.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+def mysql_estimate_row_count(model_cls, db):
+    """ Uses the information_schema to retrieve the row count for a table. """
+    query = "SELECT table_name, table_rows FROM information_schema.tables WHERE table_schema = DATABASE() AND table_name = %s"
```
Comment (jakedt) [inline data/estimate.py]: you don't need to select out the table name here if you're using an `=` where clause

File: data/estimate.py (hunk 1 above)
Comment (josephschorr) [reply data/estimate.py]: It will and I am, as it shouldn't happen in a working database

File: data/estimate.py (hunk 2 above)
Comment (josephschorr) [reply data/estimate.py]: Good point.

**Pull request merged**